### PR TITLE
feat: kcb integration

### PR DESCRIPTION
**Pull request by amosmachora**
This PR adds the logic to integrate facilities using KCB paybills to integrate with the mpesa payment aggregator.

### CODE REVIEW

**Comment by amosmachora** [comment]
@Murithijoshua this PR breaks a lot of the configuration so we might need to re-configure some JSONs.

**Comment by Murithijoshua** [comment]
@amosmachora lets chat on slack

**Comment by ojwanganto** [comment]
Thanks, @amosmachora. We have a facility to test with.

**Comment by Murithijoshua** [comment]
@amosmachora Change is effected on backend @ojwanganto you can merge this one

**Review by Murithijoshua — APPROVED** [review]

File: app/api/stk-push-callback/route.ts [hunk 1]
```diff
@@ -37,7 +36,8 @@ export const POST = async (req: NextRequest, res: NextResponse) => {
   const received: STKPushSuccessfulCallbackBody = await req.json();
 
   const origin = req.headers.get("origin") ?? "";
-  // const isAllowedOrigin = [...allowedOrigins, ...safaricomOrigins].includes(
+  // TODO limit what can send here.
+  // const isAllowedOrigin = safaricomOrigins.includes(
```
Comment (ojwanganto) [inline app/api/stk-push-callback/route.ts:40]: Do we still need the commented out lines?

**Review by ojwanganto — APPROVED** [review]

File: app/api/stk-push-callback/route.ts (hunk 1 above)
Comment (amosmachora) [reply app/api/stk-push-callback/route.ts:40]: I figured we might need them in the future so I just commented them out.

**Pull request merged**